Dockerfile

```diff
@@ -20,10 +20,11 @@ RUN apt-key adv --keyserver hkp://pgp.mit.edu:80 --recv-keys 573BFD6B3D8FBC64107
 
 WORKDIR /
 COPY feed-ingress /
+
 RUN mkdir /nginx
-RUN chown nginx:nginx /nginx
+COPY nginx.tmpl /nginx/
+RUN chown -R nginx:nginx /nginx
```

Comment:
nginx.tmpl was owned by root before, I changed this so I could edit the template in a running container (to test specifically this PR).